enhancements/baremetal/enable-baremetal-on-other-platforms.md

```diff
@@ -0,0 +1,179 @@
+---
+title: enable-baremetal-on-other-platforms
+authors:
+  - "@asalkeld"
+  - "@sadasu"
+reviewers:
+  - TBD
+  - "@hardy"
+  - "@romfreiman"
+  - "@dhellman"
+approvers:
+  - TBD
+creation-date: 2021-08-20
+last-updated: 2021-08-20
+status: implementable
+see-also:
+  - "/enhancements/baremetal-provisiong-config.md"
+replaces:
+superseded-by:
+---
+
+# Enable Baremetal on other Platforms to support ZTP
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+BMO is only available when deploying an OpenShift cluster with the baremetal
+platform (via the IPI or AI workflow). Having the ability to manage baremetal nodes from
+clusters without requiring the cluster to be on baremetal would be beneficial
+to customers.
+
+## Motivation
+
+An initial driver of this feature are the centralized host management use cases
+in edge topologies, which without this feature, is restricted to having the
+central OpenShift cluster deployed on bare metal.
+
+### Goals
+
+The specific goals of this proposal are to:
+
+Support the ZTP use case by partially enabling cluster-baremetal-operator on the following platforms:
```

Comment:
Is there another enhancement document describing the ZTP use case?

Reply:
not sure, I'll look @dhellmann

Reply:
I think it's covered in https://github.com/openshift/enhancements/blob/master/enhancements/installer/agent-based-installation-in-hive.md and also described in https://github.com/openshift/assisted-service/tree/master/docs/hive-integration

I'm not sure if we've used the ZTP acronym elsewhere in those docs though, so it might be more consistent to just refer to running "hub" clusters for multi-cluster management on other platforms (ZTP is only one possible use-case for that design)

Reply:
thanks @hardys , I'll update with the links and s/ZTP/centralized host management/